aiidalab_widgets_base/computational_resources.py

```diff
@@ -871,10 +883,17 @@ def _configure_computer_ssh(self, computer: orm.Computer, user: orm.User):
                 message = "SSH username is not provided"
                 raise RuntimeError(message) from exc
 
-        if "proxycommand" in sshcfg:
-            authparams["proxy_command"] = sshcfg["proxycommand"]
-        elif "proxyjump" in sshcfg:
-            authparams["proxy_jump"] = sshcfg["proxyjump"]
+        if "proxy_jump" in self.computer_configure:
+            authparams["proxy_jump"] = self.computer_configure["proxy_jump"]
+
+        if "proxy_command" in self.computer_configure:
+            authparams["proxy_command"] = self.computer_configure["proxy_command"]
+
+        if "key_filename" in self.computer_configure:
+            authparams["key_filename"] = self.computer_configure["key_filename"]
+
+        if "key_policy" in self.computer_configure:
+            authparams["key_policy"] = self.computer_configure["key_policy"]
```

Comment:
Such if clause can be simplified and moved to the `authparams`, but since there are still three sources for the setup, maybe good to keep it as it is?